pkg/machinepool/machinepool.go

```diff
@@ -1323,7 +1325,7 @@ func getMachinePoolReplicas(cmd *cobra.Command,
 				Default:  minReplicas,
 				Required: replicasRequired,
 				Validators: []interactive.Validator{
-					minReplicaValidator(multiAZ, false),
+					minReplicaValidator(multiAZ, false, false),
```

Comment:
Shouldn't this be `cluster.Hypershift().Enabled()`? same for the ones below it

Reply:
hmm I see, it is within a classic only function. ok

Reply:
@gdbranco There isn't a cluster for this func, would you rather me pass that into the args here? This func is used for machinepools only, so it's safe to say it's not hypershift (`Hypershift().Enabled()` will always be false)